backend/app/api/endpoints/knowledge.py

```
@@ -1327,3 +1329,148 @@ def _update_kb_summary_after_deletion(kb_id: int, user_id: int, user_name: str):
         logger.info(f"[KnowledgeAPI] KB summary update task completed: kb_id={kb_id}")
+
+
+# ============== Document Chunks Endpoints ==============
+
+
+@document_router.get("/{document_id}/chunks", response_model=DocumentChunksResponse)
+def get_document_chunks(
+    document_id: int,
+    page: int = Query(default=1, ge=1, description="Page number"),
+    page_size: int = Query(default=20, ge=1, le=100, description="Page size"),
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.13)

Unused function argument: `current_user`

(ARG001)